Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -80,9 +80,28 @@ to be used:
 :file:`public/`
 ---------------
 
-We assume here that your web root points to a folder called :file:`public` in
-a Composer-based installation as is commonly done.
-Otherwise, replace  :file:`public` with the path to your web root.
+This folder contains all files that are publicly available. Your webserver's
+web root **must** point here.
+
+This folder contains the main entry script :file:`index.php` created by Composer
+and might contain publicly available files like a :file:`robots.txt` and
+files needed for the server configuration like a :file:`.htaccess`.
+
+If required, this directory can be renamed by setting `extra > typo3/cms > web-dir`
+in the composer.json, for example to :file:`web`:
+
+..  code-block:: json
+    :caption: composer.json
```

Comment:
```suggestion
    :caption: composer.json

```

The snippet is not rendered, probably because of missing empty line after caption.